### PR TITLE
Pull request with type [UPDATE] for user [bryand-recastsoftware] and object type [ConsoleExtension] - titled [Right Click Tools]

### PR DESCRIPTION
**Pull request by bryand-recastsoftware**
Automated pull request from the Endpoint Configuration Manager Community hub.

### CODE REVIEW

**Review by sihlamMSFT — APPROVED** [review]

File: CODEOWNERS
Comment (silveyy) [inline CODEOWNERS]: I'm not seeing that the recast\community-hub team is a required reviewer.... Wondering if this needs to actually exist FIRST in the branch or if out of organization teams don't work for CODEOWERS?

File: objects/ConsoleExtension/Right Click Tools.json
Comment (silveyy) [inline objects/ConsoleExtension/Right Click Tools.json:2]: >downloadLocation [](start = 3, length = 16)

@kmyrup Last thing before we sign off on this we may want to finish your final checkin for the detonation checking and have it run on this PR. At that point I think we might be good.

File: CODEOWNERS
Comment (silveyy) [reply CODEOWNERS]: @bryand-recastsoftware  Maybe once we are all signed off and ready for submission one of us can try to complete the PR to verify this behavior. Want to make sure we have the right controls in place for Extension owners so you are in control.

---
In reply to: [610234166](https://github.com/microsoft/configmgr-hub/pull/101#discussion_r610234166) [](ancestors = 610234166)

Comment (sihlamMSFT) [reply CODEOWNERS]: My assumption is also it has to be on master branch already for it to take effect.

Comment (bryand-recastsoftware) [reply CODEOWNERS]: @silveyy: Do you want me to submit a separate PR to master for CODEOWNERS that the team approves and merges first?

Comment (sihlamMSFT) [reply CODEOWNERS]: Except the file /object/ConsoleExtension/"Right Click Tools.json" doesn't exists yet on the separate PR, but may be it will still work.

Comment (silveyy) [reply CODEOWNERS]: That or we can read the manual... on second thought let's do that. :)

---
In reply to: [610239296](https://github.com/microsoft/configmgr-hub/pull/101#discussion_r610239296) [](ancestors = 610239296)

Comment (bryand-recastsoftware) [reply CODEOWNERS]: And here I thought the plan was to just keep failing upwards.  Man, you guys and your 'reading'. 'Manuals'. Overachievers the whole the lot of you!

**Review by silveyy — APPROVED** [review]
:shipit:

**Comment by bryand-recastsoftware** [comment]
And the peasants rejoiced!

Thanks @sihlamMSFT and @silveyy for all your work and help shepherding this along!  Just updated our TP environment flawlessly.  Now to see if our code actually works.

**Comment by silveyy** [comment]
Fantastic! thanks for confirming. Quick question, did you happen to verify the "upgrade" scenario of your extension? For example had the previous version installed now new version is available in the hub.... download and update in console?

We are hopefully in the home stretch here!

**Comment by bryand-recastsoftware** [comment]
Yup, the TP had the existing release installed already.  Got the 'Download Update' button, approved install, installed (locally) and confirmed that 4.6.2103 is installed.

**Comment by silveyy** [comment]
Excellent, thanks for confirming!

**Pull request merged**
